android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.kt

```diff
@@ -66,13 +67,15 @@ class BOINCActivity : AppCompatActivity() {
         override fun onServiceConnected(className: ComponentName, service: IBinder) {
             // This is called when the connection with the service has been established, getService returns
             // the Monitor object that is needed to call functions.
-            monitor = IMonitor.Stub.asInterface(service)
+            imonitor = IMonitor.Stub.asInterface(service)
```

Comment:
If this nit used anywhere in this class, maybe it's better to move it completely to `MonitorAsync`?

Reply:
Not sure. I will leave it to the end

Reply:
fix